cli/remotebazel/remotebazel.go

```diff
@@ -54,6 +54,7 @@ const (
 var (
 	execOs            = flag.String("os", "linux", "If set, requests execution on a specific OS.")
 	execArch          = flag.String("arch", "amd64", "If set, requests execution on a specific CPU architecture.")
+	runnerImage       = flag.String("runner_image", "", "If set, requests execution on a specific runner image. Otherwise uses the default hosted runner version.")
```

Comment:
Should we call this `container_image` for better consistency with the `container-image` exec property + `buildbuddy.yaml` config field?

Reply:
This was marked resolved - forgot to push?